packages/extension-chakra-storefront/src/pages/account/addresses.jsx

```diff
@@ -348,14 +349,7 @@ const AccountAddresses = () => {
                                     removeBtnLabel={removeLabel}
                                 >
                                     {address.preferred && (
-                                        <Badge
-                                            position="absolute"
-                                            fontSize="xs"
-                                            right={4}
-                                            variant="solid"
-                                            bg="gray.100"
-                                            color="gray.900"
-                                        >
```

Comment:
Great job on moving this to theme file. Are these styling apply to all badges? If so I suggest we do a `base` styling instead of `'default' variant.